dbaas-lib/engines/k8s-pxc/controller.go

```diff
@@ -249,3 +256,49 @@ func generatePass() ([]byte, error) {
 
 	return b, nil
 }
+
+func (p *PXC) checkClusterPods(name string, st *k8sStatus) error {
+	for i := 0; i < int(st.Status.PXC.Size); i++ {
+		podData, err := p.cmd.GetObject("pod", name+"-pxc-"+strconv.Itoa(i))
+		if err != nil && !strings.Contains(err.Error(), "Not found") {
```

Comment:
I feel like "Not found" detection should be part of `cmd.GetObject` so cmd.GetObject should return some "typed" error

Reply:
https://github.com/Percona-Lab/percona-dbaas-cli/pull/101/commits/600ae3a23e647cb3f5ddb2e64960c6ca1f41050f